docs/appendices/release-notes/unreleased.rst

```diff
@@ -65,6 +65,12 @@ None
 Changes
 =======
 
+- Replaced the ``Nashorn`` by ``GraalVM`` JavaScript engine in the JavaScript
+  :ref:`user-defined functions <sql_administration_udf>` implementation.
+  This change improves the performance of the JavaScript user-defined
+  functions and upgrades the ``ECMAScript`` specification version from ``5.1``
+  to ``10.0``.
```

Comment:
how about:

> Replaced the ``Nashorn`` JavaScript engine with ``GraalVM`` for JavaScript
  :ref:`user-defined functions <sql_administration_udf>`. This change improves the performance of the JavaScript user-defined functions and upgrades ECMAScript support from ``5.1`` to ``10.0``.

(slightly reworded first line and last line)